util/Hexagon/instrumented_generator.h

```diff
@@ -0,0 +1,246 @@
+#pragma once
+
+/** \file
+ * Defines a Generator interface which injects timing code around loops and
+ * emits a report to logcat via adsprpc (Hexagon only).
+ */
+
+#include "Halide.h"
```

Comment:
There needs to be an `INTERFACE` target for this that depends on `Halide::Halide`.